template/modal/window.html

```diff
@@ -1,3 +1,3 @@
 <div modal-render="{{$isRendered}}" tabindex="-1" role="dialog" class="modal fade" ng-class="{in: animate}" ng-style="{'z-index': 1050 + index*10, display: 'block'}" ng-click="close($event)">
-    <div class="modal-dialog" ng-class="{'modal-sm': size == 'sm', 'modal-lg': size == 'lg'}"><div class="modal-content" modal-transclude></div></div>
+    <div class="modal-dialog" ng-class="size ? 'modal-' + size"><div class="modal-content" modal-transclude></div></div>
```

Comment:
Whoops, should be fixed now. You'd think after writing it right in the issue conversation I wouldn't make that sort of mistake...